### PR TITLE
Added host A check for Net::DNS MX routine

### PR DESCRIPTION
**Pull request by tmorehouse**
Added back fallback to check for valid A record for host if valid MX Record not found via Net:DNS:mx

### CODE REVIEW

**Comment by tmorehouse** [comment]
fixes #30

File: lib/Email/Valid.pm [hunk 1]
```diff
@@ -157,6 +158,15 @@ sub _net_dns_query {
     }
   }
 
+  # Chceck for A record for $host
+  my @a_rrs = Net::DNS::rr($Resolver, $host, 'A');
```
Comment (pali) [inline lib/Email/Valid.pm]: Should not you check also for AAAA record?

Comment (tmorehouse) [inline lib/Email/Valid.pm]: The original MX and Host checks only looked for A records.

I suppose assing in AAAA would be a future proof check as well for IPV4.

Comment (tmorehouse) [inline lib/Email/Valid.pm]: Although if your local SMTP server that you are using can't handle IPV6, then testing both A and AAAA might not be good if the domain resolves to only an AAAA record.

Maybe AAAA should be an optional parameter to enable IPV6 checks.

By default, Net::DNS searches default to "A IN"

**Comment by rjbs** [comment]
Thanks, applied as bf66317ea34008273314dd83f8c795eabc7bb6c3, and created #33.

**Pull request closed without merging**